.cirrus.yml

```diff
@@ -344,14 +344,8 @@ static_alt_build_task:
         ALT_NAME: 'Static build'
         # Do not use 'latest', fixed-version tag for runtime stability.
         CTR_FQIN: "docker.io/nixos/nix:2.3.6"
-    # This is critical, it helps to avoid a very lengthy process of
-    # statically building every dependency needed to build podman.
-    # Assuming the dependency and build description hasn't changed,
-    # this cache ensures only the static podman binary is built.
```

Comment:
I think this comment is helpful for developers, who may not be privy to the history and detailed workings of this task.  Would you mind adding a similar comment, highlighting the role/importance of cache in this task?

Reply:
Sure, moved the comment to the `CACHIX_AUTH_TOKEN` variable definition.